packages/components/radios/README.md

```diff
@@ -502,6 +647,7 @@ The radios Nunjucks macro takes the following arguments:
 | **items.{}.checked** | boolean   | No        | If true, radio will be checked. |
 | **items.{}.disabled** | boolean   | No        | If true, radio will be disabled. |
 | **items.{}.attributes** | object   | No        | Any extra HTML attributes (for example data attributes) to add to the radio input tag. |
+| **items.{}.conditional.{}.html** | string   | No        | HTML to be displayed when the radio is checked |
```

Comment:
Agreed... think thats worth us looking at. I couldn't find any equivalent on Gov to compare and no issues raised on NHS so up to our preference I guess

Reply:
I'm struggling to find examples because I don't know what to google for, but I'm fairly sure the convention is to use `.[].` for an array.
Anyway, it's not within the scope of this PR

Reply:
You do still need to change it from `items.{}.conditional.{}.html` to `items.{}.conditional.html` though